### ISSUE TITLE
Support js() mode for mixes

### ISSUE THREAD

**Issue by tadatuta** [bem/bem-xjst#147]
``` js
bemhtml.compile(function() {
    block('b1').js()({ blah: 123  });
    block('b2').js()({ ololo: 'trololo' });
});

bemhtml.apply({
    block: 'b1',
    js: { zzzz: 555 },
    mix: { block: 'b2', js: { qqq: 'qweqwewqeq' } }
});
```

Resulted with

``` html
<div class="b1 b2 i-bem" data-bem='{"b1":{"zzzz":555,"blah":123},"b2":{"qqq":"qweqwewqeq"}}'></div>
```

but

``` html
<div class="b1 b2 i-bem" data-bem='{"b1":{"zzzz":555,"blah":123},"b2":{"qqq":"qweqwewqeq", "ololo": "trololo"}}'></div>
```

expected.

**Comment by zxqfox** [comment]
Why? Do we start executing templates for mixes?

**Comment by tadatuta** [comment]
@zxqfox https://github.com/bem/bem-core/blob/v2/common.blocks/i-bem/i-bem.bemhtml#L393-L457 ;)

**Comment by blond** [comment]
@tadatuta how i can replace js mode and ignore input BEMJSON?

**Comment by arikon** [comment]
Should we run template for mixes?

**Comment by zxqfox** [comment]
@blond There is the same question about tag or even attrs.style.

```js
block('a')(
  tag().force()('qwe'),
  attrs()(function() {
    return {style: 'font-size: 40px !important!important!important!important'};
  })
);
// there is no way!
{ block: 'a', tag: 'non-qwe', attrs: { style: 'font-size: 42em' } }
```

etc.

**Comment by blond** [comment]
@zxqfox +1 for `force()`

**Comment by zxqfox** [comment]
@blond 
![image](https://cloud.githubusercontent.com/assets/677518/11939422/19baf546-a833-11e5-8751-e2543edb58f3.png)

**Comment by miripiruni** [comment]
Closed in favor of https://github.com/bem/bem-xjst/issues/340: Apply templates for mixes.

**Comment by miripiruni** [comment]
Closed in favor of #241
